android/gradle.properties

```diff
@@ -1,4 +1,4 @@
 ReactNativeNetInfo_compileSdkVersion=28
 ReactNativeNetInfo_buildToolsVersion=28.0.3
 ReactNativeNetInfo_targetSdkVersion=27
-ReactNativeNetInfo_minSdkVersion=16
+ReactNativeNetInfo_minSdkVersion=21
```

Comment:
Is this a breaking change? I'm not sure how this variable is actually used but I don't see any code changes in the Android code that actually require this bump. Is it your intention to remove compatibility with API16-20?

Reply:
React native [Remove support of Android API levels 16 through 20](https://github.com/react-native-community/releases/blob/master/CHANGELOG.md#android-specific-7). 
Why we should support this API level?) 
If facebook cant do this - we cant do this too :D

+  I don't remember after which version they drop compatibility with API16-20 -> You cant build your React Native(>0.64?)  app with library with `minSdkVersion < 21` ->  you will gets error:o

Reply:
It is not important if you can't build react-native 0.64 with minSdkVersion 16.
The question is: can you build our *lowest-supported react-native version* with minSdkVersion 16.

Nothing in this PR has dropped support for lower react-native versions, and nothing in this PR has dropped support for lower Android APIs, so what you are doing with this change is arbitrarily taking it upon yourself to cut off users of lower APIs, though there is no technical reason to do so.

That's user unfriendly, and it is unrelated to either removing JCenter or updating the example.

minSdkVersion 16 was supported in react-native 0.59 which is (as mentioned in another comment) still supported by the code in this repo. It wasn't bumped to minSdkVersion 21 until react-native 0.64.

I do not support this breaking change as part of this PR

Reply:
This variable will still be overwritten [here](https://github.com/react-native-netinfo/react-native-netinfo/blob/master/android/build.gradle#L35) from root project:o

But I returned ReactNativeNetInfo_minSdkVersion back as it was before me